decidim-debates/spec/commands/decidim/debates/admin/create_debate_spec.rb

```diff
@@ -58,14 +61,9 @@
       end
     end
 
-    it "sets the scope" do
-      subject.call
-      expect(debate.scope).to eq scope
-    end
-
-    it "sets the category" do
+    it "sets the taxonomies" do
       subject.call
-      expect(debate.category).to eq category
+      expect(debate.taxonomizations).to match_array(form.taxonomizations)
     end
```

Comment:
in `decidim-budgets/spec/commands/decidim/budgets/admin/create_project_spec.rb` you add some example of `context "when no taxonomizations are set" do`

```suggestion
    end
    
    context "when no taxonomizations are set" do
      let(:taxonomizations) { [] }

      it "taxonomizations are empty" do
        subject.call

        expect(debate.taxonomizations).to be_empty
      end
    end
```